### PR TITLE
fix: downgrade skopeo log msgs to warning level from error level

### PR DESCRIPTION
**Pull request by jaideepr97**
Change skopeo error messages to warning when recommending a version upgrade or unable to determine version. Only emit error logs if skopeo is not installed on the system

**Checklist:**

- [x] **Commit Message Formatting**: Commit titles and messages follow guidelines in the
  [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
- [ ] [Changelog](https://github.com/instructlab/instructlab/blob/main/CHANGELOG.md) updated with breaking and/or notable changes for the next minor release.
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.

### CODE REVIEW

File: src/instructlab/model/download.py [hunk 1]
```diff
@@ -341,10 +341,10 @@ def check_skopeo_version():
         if version.parse(installed_version) < version.parse(
             _RECOMMENDED_SCOPEO_VERSION
         ):
-            logger.error(
+            logger.debug(
```
Comment (nathan-weinberg) [inline src/instructlab/model/download.py:344]: This should be a warning
```suggestion
            logger.warning(
```

File: src/instructlab/model/download.py [hunk 2]
```diff
@@ -341,10 +341,10 @@ def check_skopeo_version():
...
                 f"skopeo version {installed_version} is lower than {_RECOMMENDED_SCOPEO_VERSION}. Consider upgrading. Downloading the model might fail."
             )
     else:
-        logger.error(
+        logger.warn(
```
Comment (nathan-weinberg) [inline src/instructlab/model/download.py:348]:
```suggestion
        logger.warning(
```

**Review by nathan-weinberg — APPROVED** [review]
Pending the CI passing this LGTM - thanks @jaideepr97!

File: src/instructlab/model/download.py (hunk 1 above)
Comment (jaideepr97) [reply src/instructlab/model/download.py]: updated, thanks

**Review by bcrochet — APPROVED** [review]
/lgtm

**Review by booxter — APPROVED** [review]

**Review by alinaryan — APPROVED** [review]

**Pull request merged**